packages/blade/src/components/Alert/__tests__/Alert.native.test.tsx

```diff
@@ -1,5 +1,73 @@
+import { fireEvent } from '@testing-library/react-native';
+
+import { Alert } from '..';
+import renderWithTheme from '~src/_helpers/testing/renderWithTheme.native';
+
 describe('<Alert />', () => {
-  it.skip('should render', () => {
-    // todo
+  it('should render', () => {
```

Comment:
Should we add snapshot checks for all the intents, borderless, & full width? Don't see any tests for those

Reply:
+1, Should at least add tests for the two different variants - borderless & fullWidth.

Reply:
I feel it creates too many snapshots to keep sane 😅, added couple of more variations in https://github.com/razorpay/blade/pull/670/commits/e98af4b3765a48d2ed6c84ab8387b32c441f6778